_posts/2018-08-22-engineering-guiding-principles.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: epic
+title: "Engineering Guiding Principles"
+date: 2018-08-22
+author: [ash]
+categories: [concepts, teams, engineering, culture, meta]
+---
+
+The Artsy Engineering team recently underwent the process of defining our guiding principles; you can read through the pull request [here][pr] and the finished principles [here][principles]. In this blog post, I'd like to use our experience of defining these to answer the following questions:
+
+- Why define engineering guiding principles?
+- What makes principles different from company values?
+- How to define guiding principles?
+
+Let's dive in.
+
+<!-- more -->
+
+## Why Define Engineering Principles?
+
+Artsy's CEO has a [blog post][company_values] that discusses why you should define company values and, unsurprisingly, a lot of the same rationale applies to an engineering team's guiding principles. In summary:
+
+- Undefined principles lead to cultural debt (similar to technical debt, but in terms of a team's culture).
+- Defined values lead to greater empowerment (engineers feel safe making decisions on their own, based on the principles).
+- Defined values reduce the risk of unconscious bias (we help our decisions get made consistently).
+- Values are your brand (less applicable for Artsy Engineering; our [contributions][gh] to the software industry are our brand).
```

Comment:
I don't think eng values are less applicable for our brand than other companies -- our open source contributions are the biggest part of our brand but our values could definitely add to that.

Reply:
This is an interesting idea, maybe it's on the lines of: "yes our OSS contributions are basically our brand, but the engine that powers them are our eng values"